src/ucs/datastruct/string_set.h

```diff
@@ -26,8 +26,10 @@ typedef khash_t(ucs_string_set) ucs_string_set_t;
 /*
  *  Check if ucs_string_set_t is empty
  */
-#define ucs_string_set_is_empty(_string_set_ptr) \
-    (kh_size(_string_set_ptr) == 0)
+static UCS_F_ALWAYS_INLINE int ucs_string_set_is_empty(ucs_string_set_t *sset)
```

Comment:
1. can it be const ptr?
2. usually inline functions are defined after the non-inline ones